### PR TITLE
Leader's Den Other Clan Fail Outcomes

### PR DESCRIPTION
**Pull request by tobes03**
## About The Pull Request

14 new outcomes for every possible failure! Now you can fail in STYLE. Many of them are not stat- or trait-locked.

## Why This Is Good For ClanGen

lil variation, eh.

## Proof of Testing
image

image

Here are a couple samples

### CODE REVIEW

**Comment by tobes03** [comment]
Couple typos need to be cleared up, don't merge yet

File: resources/dicts/events/leader_den/fail/other_clan.json [hunk 1]
```diff
@@ -18,97 +58,256 @@
...
+    "rel_change": -1,
+    "m_c": { 
+      "skill": [ "SPEAKER,1" ]
+    },
+    "player_clan_temper": ["any"],
+    "other_clan_temper": ["any"]
+  },
+  {
+    "interaction_type": "praise",
+    "event_text": "o_c_n's cats keep their distance from c_n during the Gathering. No matter what tack m_c tries, o_c_n's leader doesn't seem receptive. Perhaps this alliance is weakning for reasons {PRONOUN/m_c/subject} {VERB/m_c/are/is}n't aware of.",
```
Comment (scribblecrumb) [inline resources/dicts/events/leader_den/fail/other_clan.json]:
```suggestion
    "event_text": "o_c_n's cats keep their distance from c_n during the Gathering. No matter what tack m_c tries, o_c_n's leader doesn't seem receptive. Perhaps this alliance is weakening for reasons {PRONOUN/m_c/subject} {VERB/m_c/are/is}n't aware of.",
```

File: resources/dicts/events/leader_den/fail/other_clan.json [hunk 2]
```diff
@@ -18,97 +58,256 @@
...
+    "m_c": {
+      "trait": [ "childish" ]
+     },
+    "player_clan_temper": ["any"],
+    "other_clan_temper": ["any"]
+  },
+  {
+    "interaction_type": "praise",
+    "event_text": "At the Gathering, o_c_n's leader seems much more interested in chatting with the other leaders than m_c. When {PRONOUN/m_c/subject} {VERB/m_c/try/tries} to include {PRONOUN/m_c/self}, {PRONOUN/m_c/subject} just {VERB/m_c/draw/draws} awkward looks from the other leaders.",
+    "rel_change": 2,
```
Comment (scribblecrumb) [inline resources/dicts/events/leader_den/fail/other_clan.json]:
```suggestion
    "rel_change": -2,
```

File: resources/dicts/events/leader_den/fail/other_clan.json [hunk 3]
```diff
@@ -18,97 +58,256 @@
...
+    "m_c": {
+      "trait": [ "lonesome", "nervous", "insecure", "gloomy", "grumpy" ]
+     },
+    "player_clan_temper": ["any"],
+    "other_clan_temper": ["any"]
+  },
+  {
+    "interaction_type": "praise",
+    "event_text": "m_c goes on and on about what a great alliance c_n and o_c_n have during {PRONOUN/m_c/poss} speech at the Gathering. o_c_n's leader coughs meaningfully; during their speech, they announce o_c_n will be focusing on its own priorities in the coming moons and suggest c_n does the same.",
+    "rel_change": 2,
```
Comment (scribblecrumb) [inline resources/dicts/events/leader_den/fail/other_clan.json]:
```suggestion
    "rel_change": -2,
```

**Review by scribblecrumb — CHANGES_REQUESTED** [review]
This is such a crazy amount of new outcomes, ur so cool tobes <3

**Review by scribblecrumb — APPROVED** [review]

**Pull request merged**